examples/webgl_postprocessing_backgrounds.html

```diff
@@ -15,8 +15,20 @@
 		<div id="container"></div>
 
 		<script type="module">
-
-			import * as THREE from '../build/three.module.js';
```

Comment:
I think @mrdoob prefers this pattern. Compare with the existing examples.

Reply:
Indeed, we actually moved from named imports to this style 👍

Reply:
Sure, done.